scripts/test_release_summary.py

```
@@ -366,6 +369,295 @@ def test_build_gates_takes_worst_gate():
+    assert len(areas) == 1
+    assert areas[0]["definition"]["name"] == "feature-element-merge"
+    assert areas[0]["paths"] == ["apps/aegroweb/src/app/presentation/edm/"]
+
+
+def test_load_gated_areas_survives_malformed_file(tmp_path):
+    (tmp_path / ".github").mkdir()
+    (tmp_path / ".github/toggle-gated-areas.json").write_text("{ nao json", encoding="utf-8")
+    assert rs.load_gated_areas(str(tmp_path), rs.parse_toggle_registry(TS_REGISTRY)) == []
+
+
+def test_inherited_area_never_claims_live(tmp_path):
```

Ruff (0.16.1)

[warning] Unused function argument: `tmp_path`

(ARG001)